cluster-setup/base/performance/performance_profile.yaml

```diff
@@ -10,3 +10,5 @@ spec:
     defaultHugepagesSize: "1G"
   realTimeKernel:
     enabled: true
+  kubeletConfig:
+    topologyManager: "single-numa-node"
```

Comment:
I consider kubeletConfig and topologyManager to be implementation details. So I would change the structure and naming to something like:

```
numa:
  default-policy: "single-numa-node"
```

This would allow us to change the underlying implementation in the future. For example, our plan is to see if per-pod policy can be implemented. My naming (although probably not perfect) would still make sense in that case.

Reply:
+1
Please don't expose the complete KubeletConfig. If you need all of it, you can use it directly, the performance operator wouldn't add any value then IMHO

Reply:
I'm not fully comfortable with "default-policy". What if we still include the word "topology"? 

It could be either

```
numa:
    topology-policy: (none|best-effort|single-numa-node|restricted)
```

or perhaps:

```
topology:
    policy: (none|best-effort|single-numa-node|restricted)
```

WDYT?

Reply:
numa.topology-policy: sounds good to me.